mailpoet/tests/acceptance/Forms/SubscriptionFormCest.php

```diff
@@ -129,4 +129,22 @@ public function subscriptionAfterDisablingConfirmation(\AcceptanceTester $i) {
     $i->waitForText("You’ve been successfully subscribed to our newsletter!", self::CONFIRMATION_MESSAGE_TIMEOUT, '.mailpoet_validate_success');
     $i->seeNoJSErrors();
   }
+
+  public function subscriptionNewPageConfirmation(\AcceptanceTester $i) {
+    $i->wantTo('Subscribe to a form and to see new page confirmation');
+    $i->login();
+    $i->amOnMailpoetPage('Forms');
+    $i->clickItemRowActionByItemName(self::FORM_NAME, 'Edit');
+    $i->waitForElement('[data-automation-id="form_title_input"]');
+    $i->click('(//div[@class="components-radio-control__option"])[2]'); // Click Go to Page option
+    $i->selectOption('.components-select-control__input', 'Sample Page');
+    $i->saveFormInEditor();
+    $i->amOnPage('/form-test');
+    $i->executeJS('window.scrollTo(0, document.body.scrollHeight);');
+    $i->switchToIframe('#mailpoet_form_iframe');
+    $i->fillField('[data-automation-id="form_email"]', $this->subscriberEmail);
+    $i->scrollTo('.mailpoet_submit');
+    $i->click('.mailpoet_submit');
+    $i->waitForText('Sample Page');
```

Comment:
@veljkho As a future improvement, we could check if the redirection happens outside the frame, not inside it.